milkdown-editor/src/style.css

```
@@ -0,0 +1,91 @@
+  --crepe-color-primary: #b5b5b5;
+  --crepe-color-secondary: #4d4d4d;
+  --crepe-color-on-secondary: #d6d6d6;
+  --crepe-color-inverse: #e5e5e5;
+  --crepe-color-on-inverse: #2a2a2a;
+  --crepe-color-inline-code: #ff6666;
+  --crepe-color-error: #ff6666;
+  --crepe-color-hover: #232323;
+  --crepe-color-selected: #2f2f2f;
+  --crepe-color-inline-area: #2b2b2b;
+
+  --crepe-font-title: 'Noto Serif', Cambria, 'Times New Roman', Times, serif;
```

Stylelint (17.9.0)

[error] Expected "Cambria" to be "cambria" (value-keyword-case)

(value-keyword-case)